src/main/java/ru/svgogin/service/spark/handler/RestResponseEntityExceptionHandler.java

```diff
@@ -35,4 +38,32 @@
     return handleExceptionInternal(ex, bodyOfResponse,
         new HttpHeaders(), HttpStatus.NOT_FOUND, request);
   }
+
+  @ExceptionHandler(value = {ConstraintViolationException.class })
+  protected ResponseEntity<Object> handleNotValid(RuntimeException ex, WebRequest request) {
+    String message = ex.getMessage();
+    var bodyOfResponse = new ErrorDto(ErrorDto.ErrorCode.ERROR003, message);
+    log.warn(message);
+    return handleExceptionInternal(ex, bodyOfResponse,
+        new HttpHeaders(), HttpStatus.BAD_REQUEST, request);
+  }
+
+  @NonNull
+  @Override
+  protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
+                                                                @NonNull HttpHeaders headers,
+                                                                @NonNull HttpStatus status,
+                                                                @NonNull WebRequest request) {
+    var errorField = ex.getBindingResult().getFieldErrors().iterator().next().getField();
+    var message = "Invalid format of request parameter: " + errorField;
+    //toDO If a parameter is not passed in the request, I don't know,
```

Comment:
please, add tests and also for this case

Reply:
Done